datasets/shield.md

```diff
@@ -1,5 +1,121 @@
+[this summary table](https://metrics.mozilla.com/~sguha/report/normandy_recipes.html),
+or the list of active recipes at
+https://normandy.cdn.mozilla.net/api/v1/recipe/signed/.
+
+## Tables
+
+These tables should be accessible from ATMO, Databricks, Presto, and Athena.
+
+### `experiments` column
+
+[`main_summary`](../datasets/batch_view/main_summary/reference.html),
+[`clients_daily`](../datasets/batch_view/clients_daily/reference.html),
+[`crash_summary`](../datasets/batch_view/crash_summary/reference.html),
+and some other tables
+include a `experiments` column
+which is a mapping from experiment stub to branch.
+
+You can collect rows from enrolled clients using query syntax like:
+
+```sql
+SELECT
+  *,
+  experiments['some-experiment-stub-12345'] AS branch
+FROM clients_daily
+WHERE experiments['some-experiment-stub-12345'] IS NOT NULL
+```
+
+### `experiments`
+
+The `experiments` table is a subset of rows from `main_summary`
+reflecting pings from clients that are currently enrolled in an experiment.
+The `experiments` table has additional string-type
+`experiment_id` and `experiment_branch` columns,
+and is partitioned by `experiment_id`, which makes it efficient to query.
+
+Experiments deployed to large fractions of the release channel
+may have the `isHighVolume` flag set in the Normandy recipe;
+those experiments will not be aggregated into the `experiments` table.
+
+Please note that the `experiments` table cannot be used
+for calculating retention for periods extending beyond
+the end of the experiment.
+Once a client is unenrolled from an experiment,
+subsequent pings will not be captured by the `experiments` table.
+
+### `events`
+
+The [`events` table](../datasets/batch_view/events/reference.html) includes
+Normandy enrollment and unenrollment events.
+
+Normandy events have event category `normandy`.
+The event value will contain the experiment slug.
+
+The event schema is described
+[in the Firefox source tree](https://hg.mozilla.org/mozilla-central/file/c4c125ee2556/toolkit/components/normandy/lib/TelemetryEvents.jsm#l12).
+
+The `events` table is updated daily.
+
+### `telemetry_shield_study_addon_parquet`
+
+The `telemetry_shield_study_addon_parquet` table contains SHIELD telemetry from add-on experiments,
+i.e. key-value pairs sent with the
+`browser.study.sendTelemetry()` method from the
+[SHIELD study add-on utilities](https://github.com/mozilla/shield-studies-addon-utils/)
+library.
+
+The schema for shield-study-addon pings is described in the
+[`mozilla-pipeline-schemas` repository](https://github.com/mozilla-services/mozilla-pipeline-schemas/tree/dev/schemas/telemetry/shield-study-addon).
+
+The key-value pairs are present in `data` attribute of the `payload` column.
+
+The `telemetry_shield_study_addon_parquet` table is produced by direct-to-parquet;
+data latency should be less than 1 hour.
+
+### `telemetry_shield_study_parquet`
+
+Telemetry from Normandy itself, containing enrollment and unenrollment events.
```

Comment:
Is it actually Normandy emitting these and not shield-study-utils?

Reply:
Whoops, you're right. Will Normandy still emit events that get collected to the `events` table for these?

Reply:
Yes, it will.